app/src/main/java/com/hedvig/app/util/Validation.kt

```diff
@@ -1,15 +1,17 @@
 package com.hedvig.app.util
 
 import androidx.annotation.StringRes
+import androidx.core.util.PatternsCompat
 import com.hedvig.app.R
 
 object Regexes {
-  val emailRegex = Regex("^\\S+@\\S+\$")
   val phoneNumberRegex = Regex("([+]*[0-9]+[+. -]*)")
 }
 
+fun String.isValidEmail() = isBlank() && PatternsCompat.EMAIL_ADDRESS.matcher(this).matches()
```

Comment:
One concern for extension functions like these.
This means that in every single part of the code base, whenever we have a String type and we do `.` on it, we're gonna be suggested this as a possible thing we want to use.
And the ratio of when we do `.` on a String and when we specifically have that string be an email and we might be interested in this function is quite low. In a sense this will be there but not used 95%+ (I just made that number up) of the time.

With all that said, we can go ahead and keep this if you want or we don't come up with a good alternative, but it's an interesting thing to keep in mind in general imo 😊

Reply:
Good point!